### PR TITLE
CSP Changes for Project, Observation and Survey 

### PR DESCRIPTION
**Pull request by spsac08**

### CODE REVIEW

File: cloud_storage/constants.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+status_code_1 = 201
```
Comment (adithyadinesh0412) [inline cloud_storage/constants.py]: Change to single list

File: cloud_storage/cloud.py [hunk 2]
```diff
@@ -25,57 +17,75 @@ def __init__(self):
         '''
         self.sections = config.sections()
 
-    def upload_to_cloud(self, blob_Path, local_Path, file_Name):
-        '''
-        Function to upload the file to respective cloud based on the available section in the config file
 
-        :param: blob_Path - The path where the file would be stored in the cloud 
+        
+    def upload_to_cloud(self, filesList ,folderPathName, local_Path):
```
Comment (rakeshSgr) [inline cloud_storage/cloud.py:22]: @adithyadinesh0412 function should accept array of files as input, We can implement this in next release. We need to make note of this

File: observations/pyspark_observation_status_batch.py [hunk 3]
```diff
@@ -29,12 +29,16 @@
 config = ConfigParser(interpolation=ExtendedInterpolation())
 config.read(config_path[0] + "/config.ini")
 
-sys.path.append(config.get("COMMON", "cloud_module_path"))
+# sys.path.append(config.get("COMMON", "cloud_module_path"))
```
Comment (rakeshSgr) [inline observations/pyspark_observation_status_batch.py]: @adithyadinesh0412 we can remove this.

File: observations/pyspark_observation_status_batch.py [hunk 4]
```diff
@@ -549,13 +553,39 @@ def observed_data(vts, entype):
...
+   sys.exit() 
 
 
 sl_status_spec = {}
+
+#get Druid spec from config
 sl_status_spec = json.loads(config.get("DRUID","observation_status_injestion_spec"))
+
+# updating Druid spec adding type and URI'S
+sl_status_spec["spec"]["ioConfig"]["inputSource"] = uploadResponse['inputSource']
```
Comment (rakeshSgr) [inline observations/pyspark_observation_status_batch.py]: @adithyadinesh0412 create a function and map cloud path, as we do in all the batch scripts.

**Pull request merged**